tasks/add-unique-granuleID/README.md

```diff
@@ -0,0 +1,106 @@
+# @cumulus/add-unique-granuleID
+
+This is a [Cumulus](https://nasa.github.io/cumulus) task which takes the following actions on each granule in an incoming set of payload granules:
+
+- Adds the existing `granuleId` to the `producerGranuleId` key
```

Comment:
Nit: It doesn't add it, it populates producerGranuleId with granuleId's value

Reply:
Good point - let me know what you think of 7263f4e2755